pages/1.11/release-notes/1.11.9/index.md

```diff
@@ -0,0 +1,122 @@
+---
+layout: layout.pug
+navigationTitle: Release Notes for 1.11.9
+title: Release Notes for 1.11.9
+menuWeight: 5
+excerpt: Release notes for DC/OS 1.11.9
+---
+
+DC/OS Version 1.11.9 was released on January 31, 2019.
+
+[button color="purple" href="https://downloads.dcos.io/dcos/stable/1.11.9/dcos_generate_config.sh"]Download DC/OS Open Source[/button]
+
+[button color="light" href="https://support.mesosphere.com/hc/en-us/articles/213198586"]Download DC/OS Enterprise[/button]
+
+DC/OS 1.11.9 includes the following components:
+- Apache Mesos 1.5.x [change log](https://github.com/apache/mesos/blob/9c28b265804269de9411f04cfcccefe3bdd8ef1a/CHANGELOG).
+- Marathon 1.6.567 [change log](https://github.com/mesosphere/marathon/tree/2d8b3e438).
+- Metronome 0.4.5 [change log](https://github.com/dcos/metronome/releases/tag/v0.4.5).
+
+# Release Summary
+
+DC/OS is a distributed operating system that enables you to manage resources, application deployment, data services, networking, and security in an on-premise, cloud, or hybrid cluster environment. 
+
+# Issues Fixed in DC/OS 1.11.9
+The issues that have been fixed in DC/OS 1.11.9 are grouped by feature, functional area, or component. Most change descriptions include one or more issue tracking identifiers for reference.
+
+## GUI 
+- DCOS-40640, DCOS-45803 - In system tests, we use isolation to run tests in parallel. When we reuse `node_modules` it is important to link them in an isolation directory. In version 3.x, cypress caches its binary in a different folder. It is important to link the `.cache` folder. Cypress cache installation is required to speed up the build. Therefore, an automated bump of DC/OS UI to version v1.26.2 resolves this issue. 
+
+## Installing
+- DCOS-40442 - A fix to handle `AzureException` during storage upload.
```

Comment:
This appears to have been a bug in our TeamCity (build/testing) scripts.  It is not part of the product or part of any flow a customer would use.